src/Chapter2.hs

```diff
@@ -620,7 +636,12 @@ Write a function that takes elements of a list only on even positions.
 >>> takeEven [2, 1, 3, 5, 4]
 [2,3,4]
 -}
-takeEven = error "takeEven: Not implemented!"
+takeEven :: [a] -> [a]
+takeEven [] = []
+takeEven [x] = [x]
+takeEven [x, _] = [x]
+takeEven (x:_:rest) = [x] ++ (takeEven rest)
```

Comment:
The same comment about using `:` instead of `++` as for `duplicate` :slightly_smiling_face: